src/components/AppWindow/MenuComponents/MenuButton.tsx

```diff
@@ -1,36 +1,55 @@
-import React, {useState} from 'react'
-import {HiOutlineDotsCircleHorizontal} from 'react-icons/hi'
-import {MenuModal} from './MenuModal'
+import React, { useState } from 'react'
+import { HiOutlineDotsCircleHorizontal } from 'react-icons/hi'
+import { MenuModal } from './MenuModal'
 import styled from 'styled-components'
-
-
-
+import { ShareModal } from './MenuOptions/ShareModal'
 export const MenuButton = () => {
```

Comment:
How is this a reusable component? According to the task it should be  Add a Menu button to each App with an ability to provide different menu options"